src/SprykerEco/Yves/Computop/Converter/InitCreditCardConverter.php

```diff
@@ -15,28 +15,28 @@
 class InitCreditCardConverter extends AbstractInitConverter
 {
     /**
-     * @param array $decryptedArray
+     * @param array $responseParamsArray
      * @param \Generated\Shared\Transfer\ComputopApiResponseHeaderTransfer $header
```

Comment:
Do we need this changes? It looks not related. The same question for all others below.

Reply:
It was requested by other developers while previous CR